packages/components/nodes/chatmodels/GoogleVertexAI/GoogleVertexAI.ts

```diff
@@ -0,0 +1,115 @@
+import { ICommonObject, INode, INodeData, INodeParams } from '../../../src/Interface'
```

Comment:
can we rename to file and folder name to `ChatGoogleVertextAI`, just to prevent same name as the one in `llms` folder

Reply:
of course!